.travis.yml

```diff
@@ -2,9 +2,6 @@ language: node_js
 
 matrix:
   include:
-    - node_js: "4"
-      addons:
-        postgresql: "9.1"
```

Comment:
please be aware that this change will prevent original repo to accept this change.
worth add comment here (in this pr) explaining why you removed support of node 4

Reply:
sure thing